packages/apis/Cargo.toml

```diff
@@ -6,10 +6,11 @@ version.workspace = true
 license.workspace = true
 repository.workspace = true
 authors.workspace = true
+publish.workspace = true
 
 [dependencies]
-babylon-merkle   = { path = "../merkle", version = "0.17.0" }
-babylon-proto    = { path = "../proto", version = "0.17.0" }
+babylon-merkle   = { path = "../merkle" }
```

Comment:
why not `babylon-merkle   = { path = "../merkle", version = "v1.0.0-rc.0" }`?

Reply:
same as above